.github/shared/src/changed-files.js

```diff
@@ -122,6 +241,20 @@ export function swagger(file) {
     typeof file === "string" &&
     json(file) &&
     (dataPlane(file) || resourceManager(file)) &&
-    !example(file)
+    !example(file) &&
+    !scenario(file)
+  );
+}
+
+/**
+ * @param {string} [file]
+ * @returns {boolean}
+ */
+export function scenario(file) {
```

Comment:
@raych1: What is the purpose of the "scenarios" folder?  I'm not aware of an official use in folder structure v1 or v2.

Reply:
It was used by the scenario test which we've retired. However, there're still some files haven't been cleaned up, such as [this one](https://github.com/Azure/azure-rest-api-specs/tree/4cb330be555a994ed507acf739e3349b7cb76d85/specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/stable/2022-05-01/scenarios).

Do you think we just help service team clean them up or leave them as is? From tooling perspective, breaking change tool doesn't need to run against these files.